src/app/models/recommendation.model.ts

```diff
@@ -0,0 +1,10 @@
+export interface Recommendation {
+  album: string;
+  apple_music_player_url: string;
+  artist: string;
+  embed_content: string;
+  id: number;
+  song_art_image_url: string;
+  title: string;
+  url: string;
```

Comment:
for objects coming from the server, it's good to mark each field as optional

(e.g. `url?: string`)

that way, you force yourself to ensure that the server has actually sent the data you expect